cmd/globals.go

```diff
@@ -109,6 +109,9 @@ var (
 	// Time when object layer was initialized on start up.
 	globalBootTime time.Time
 
+	globalActiveCred       credential
+	globalPublicCerts      []*x509.Certificate
+	globalObjCacheDisabled bool
```

Comment:
We would need to keep it as global because,

1. We would need to validate how multi-node minio is started.  This check needs to be done in post login (currently not).

2. I see flow to `newXLObjects()` is `serverMain()` -> `newObjectLayer()` -> `newXLObjectLayer()` -> `newXLObjects()`.  Passing `globalObjCacheDisabled` flag over these functions is not a good idea.